### PR TITLE
feat: update code generation to generate all typescript templates

### PR DESCRIPTION
**Pull request by JivusAyrus**
## Motivation and Context
To ensure the TS Operations always work.

#### Checklist

- [x] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: testapps/nextjs/.wundergraph/wundergraph.config.ts [hunk 1]
```diff
@@ -183,15 +183,14 @@ configureWunderGraphApplication({
...
-	codeGenerators: [
-		{
-			templates: [...templates.typescript.all],
-		},
-		{
-			templates: [new NextJsTemplate()],
-			path: '../components/generated',
-		},
-	],
+	generate: {
```
Comment (StarpTech) [inline testapps/nextjs/.wundergraph/wundergraph.config.ts:186]: Do you plan to update all examples, apps in this repo and org?

Comment (JivusAyrus) [reply testapps/nextjs/.wundergraph/wundergraph.config.ts:186]: for examples, its after the release right?

Comment (StarpTech) [reply testapps/nextjs/.wundergraph/wundergraph.config.ts:186]: Only to bump versions, not for migration. It'd be great to have them in this PR.

**Review by jensneuse — APPROVED** [review]

**Review by fiam — APPROVED** [review]
LGTM

**Review by StarpTech — APPROVED** [review]
LGTM

**Comment by StarpTech** [comment]
What does happen when people have defined it? As far as I remember we filter doubles right?

**Comment by JivusAyrus** [comment]
> What does happen when people have defined it? As far as I remember we filter doubles right?

Yep, it should not matter.

**Pull request merged**